src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+  background: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  transition: 0.3s;
+
```

Comment:
you need to add padding to card and then you dont need to add margin to every next block